site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,417 @@
+For those who force enabled VSync in their driver settings, Citra wasn’t able to disable VSync, which resulted in inconsistent frame limiting. 
+This was picked up by [epicboy](https://github.com/ameerj) and molded to work with multiple windows being displayed by one instance of Citra.
+
+To enable this feature, head to `View -> Screen Layout -> Separate Windows` from within Citra. Launch your game, and from there you can edit the size and location of the windows to wherever you want on your PC!
+
+{{< figure src="screens.png"
+    title="Map on one screen, gameplay on the other!" >}}
+
+### Ask for confirmation when changing games from the game list ([#6186](https://github.com/citra-emu/citra/pull/6186)) by [foghawk](https://github.com/foghawk)
+
+When you run Citra without Single Window Mode enabled, you have the game window and the game list window open at the same time. If you clicked another game in the game list, it’d instantly boot, closing your current game. 
+
+After experiencing this themselves by accidentally clicking onto another game while already in a game, [foghawk](https://github.com/foghawk) added a confirmation check. No longer will you lose any important save data from this issue, and you can carry on playing after selecting one of the two options!
+
+{{< figure src="stopEmu.png"
+    title="No more accidentally losing your progress." >}}
+	
+## Per-game configurations ([#6187](https://github.com/citra-emu/citra/pull/6187), [#6219](https://github.com/citra-emu/citra/pull/6219), [#6224](https://github.com/citra-emu/citra/pull/6224)) by [GPUCode](https://github.com/GPUCode)
+
+Based off of the yuzu implementation of the same name, Per-Game Configurations have landed in Citra! This is part of a series of changes coming to Citra to modernize the frontend of the emulator.
+
+With this new change in Citra, you no longer have to manually adjust the settings every time you launch a game. Each game now has its own individual settings profile that you can customize to your liking. You can easily modify the emulation speed, graphical settings, audio settings, and volume of each game, among many other options. This feature saves you time and hassle, allowing you to quickly jump into playing your favorite games without worrying about adjusting settings every time.
+
+With this PR, the option for **Alternate Speed**, which was available in the general settings of Citra, has been removed in favour of an option called **Per-game Emulation Speed**. This setting works in the exact same was as the previous **Alternate Speed** setting, however it is now found inside of the per-game settings instead of the global settings. The hotkey for this has also been renamed appropriately to reflect the name change of the setting.
```

Comment:
```suggestion
With this PR, the option for **Alternate Speed**, which was available in the general settings of Citra, has been removed in favour of an option called **Per-game Emulation Speed**. This setting works in the exact same way as the previous **Alternate Speed** setting, however it is now found inside of the per-game settings instead of the global settings. The hotkey for this has also been renamed appropriately to reflect the name change of the setting.
```